code/__HELPERS/global_lists.dm

```diff
@@ -111,9 +111,9 @@
 		var/datum/robolimb/R = new limb_type()
 		GLOB.all_robolimbs[R.company] = R
 		if(!R.unavailable_at_chargen)
-			if(R != "head" && R != "chest" && R != "groin" ) //Part of the method that ensures only IPCs can access head, chest and groin prosthetics.
-				if(R.has_subtypes) //Ensures solos get added to the list as well be incorporating has_subtypes == 1 and has_subtypes == 2.
-					GLOB.chargen_robolimbs[R.company] = R //List only main brands and solo parts.
+			if(R != "head" && R != "chest" && R != "groin")	//Part of the method that ensures only IPCs can access head, chest and groin prosthetics.
+				if(R.has_subtypes != 0)						//Ensures solos get added to the list as well be incorporating has_subtypes == 1 and has_subtypes == 2.
+					GLOB.chargen_robolimbs[R.company] = R	//List only main brands and solo parts.
```

Comment:
This is just spacing and clarity. Previously `R.has_subtypes` could be `null`. I removed that possibility, but I wanted to make the line of code made the most sense in the context of its associated comment.